awx/main/migrations/0136_cleanup_ee_images.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.16 on 2021-04-14 16:21
+
+from django.db import migrations
+from django.utils.timezone import now
+
+
+def create_cleanup_ee_images(apps, schema_editor):
+    SystemJobTemplate = apps.get_model('main', 'SystemJobTemplate')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    sjt_ct = ContentType.objects.get_for_model(SystemJobTemplate)
+    now_dt = now()
+    sjt, created = SystemJobTemplate.objects.get_or_create(
```

Comment:
Do (or should) these run by default on a schedule?  If so, how often cc @shanemcd @wenottingham @kdelee